lrauv_system_tests/worlds/flat_seabed_two_vehicles.sdf

```diff
@@ -0,0 +1,404 @@
+          <color>0 1 0 1</color>
+        </insert>
+        <insert>
+          <!-- 0.1 km x 0.1 km -->
+          <cell_count>100</cell_count>
+          <vertical_cell_count>0</vertical_cell_count>
+          <!-- 1 m -->
+          <cell_length>1</cell_length>
+          <pose>0 0 0  0 0 0</pose>
+          <color>0.5 0.5 0.5 1</color>
+        </insert>
+      </plugin>
+    </gui>
+
+    <light type="directional" name="sun">
+      <cast_shadows>true</cast_shadows>
+      <pose>0 0 10 0 0 0</pose>
+      <diffuse>1 1 1 1</diffuse>
+      <specular>0.5 0.5 0.5 1</specular>
+      <attenuation>
+        <range>1000</range>
+        <constant>0.9</constant>
+        <linear>0.01</linear>
+        <quadratic>0.001</quadratic>
+      </attenuation>
+      <direction>-0.5 0.1 -0.9</direction>
+    </light>
+
+    <!-- This invisible plane helps with orbiting the camera, especially at large scales -->
+    <model name="horizontal_plane">
+      <static>true</static>
+      <link name="link">
+        <visual name="visual">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <!-- 300 km x 300 km -->
+              <size>300000 300000</size>
+            </plane>
+          </geometry>
+          <transparency>1.0</transparency>
+        </visual>
+      </link>
+    </model>
+
+    <model name="sea_bottom">
+      <static>true</static>
+      <pose>0 0 -100 0 0 0</pose>
+      <link name="link">
+        <collision name="collision">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <!-- 300 km x 300 km -->
+              <size>300000 300000</size>
+            </plane>
+          </geometry>
+        </collision>
+        <visual name="visual">
+          <material>
+            <ambient>0.5 0.5 0.5</ambient>
+            <diffuse>0.5 0.5 0.5</diffuse>
+          </material>
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <!-- 300 km x 300 km -->
+              <size>300000 300000</size>
+            </plane>
+          </geometry>
+        </visual>
+      </link>
+    </model>
+
+    <include>
+      <pose>0 0 -80 0 0 0</pose>
+      <uri>tethys_equipped</uri>
+      <name>tethys</name>
+
+      <experimental:params>
+        <plugin element_id="gz::sim::systems::CommsEndpoint" action="modify">
+          <address>1</address>
+          <topic>1/rx</topic>
+        </plugin>
+      </experimental:params>
+    </include>
+
+    <include>
+      <pose>5 0 -80 0 0 0</pose>
+      <uri>tethys_equipped</uri>
+      <name>daphne</name>
+
+      <experimental:params>
+        <plugin element_id="gz::sim::systems::CommsEndpoint" action="modify">
+          <address>2</address>
+          <topic>2/rx</topic>
+        </plugin>
+        <plugin element_id="gz::sim::systems::Thruster" action="modify">
+          <namespace>daphne</namespace>
+          <joint_name>propeller_joint</joint_name>
```

Comment:
@adityapande-1995 hmm, why do we need to modify the thrusters?

Reply:
The namespace in ``tethys_equipped`` is hardcoded to ``tethys``, so by default all 3 LRAUVs listen for commands on ``/model/tethys/joint/propeller_joint/cmd_vel`` , I modified the namespace to ``daphne`` for the second one, so it listens for commands on ``/model/daphne/joint/propeller_joint/cmd_vel`` now.